db.go

```diff
@@ -461,7 +434,18 @@ func (db *DB) getBlock(id ulid.ULID) (*Block, bool) {
 	return nil, false
 }
 
-func (db *DB) reload() (err error) {
+func stringsContain(set []string, elem string) bool {
```

Comment:
Why not strings.Contain?

Reply:
That finds substrings in strings. We need to find a string in a slice.

Reply:
Facepalm. My first thought was "I wonder how often this function was written in go codebases", but then got distracted by the function name.